### PR TITLE
Add Tableau Prep specific instructions for AWS connection how-to

### PR DESCRIPTION
**Pull request by kyrasturgill**

### CODE REVIEW

**Comment by kyrasturgill** [comment]
Added this "s3://" tip because having incorrect formatting for my results bucket was giving me some grief for a bit

File: How-To/Connect-to-AWS-Resources.md [hunk 1]
```diff
@@ -142,19 +143,20 @@ As with R, enabling [unload](https://laughingman7743.github.io/PyAthena/pandas.h
     df = cursor.as_pandas()
     ```
 
-### Tableau
+### Tableau and Tableau Prep
 
 You will likely need to work with IT admins for permissions to do the following.
 
-1. Install [Tableau Desktop version 2022.1](https://www.tableau.com/support/releases/desktop/2022.1.23) if you plan to publish a file to the CCAO's Tableau Server. The version of Tableau Desktop [cannot be more recent](https://help.tableau.com/current/desktopdeploy/en-us/desktop_deploy_version_compat_top.htm) than the version of Tableau Server.
+1. Install [Tableau Desktop](https://www.tableau.com/support/releases) and [Tableau Prep](https://www.tableau.com/support/releases/prep). Verify with Kyra or Nicole which versions to install. If you plan to publish a file to the CCAO's Tableau Server, the versions of Tableau Desktop and Tableau Prep [cannot be more recent](https://help.tableau.com/current/desktopdeploy/en-us/desktop_deploy_version_compat_top.htm) than the version of Tableau Server.
```
Comment (dfsnow) [inline How-To/Connect-to-AWS-Resources.md:150]: **nitpick:** Let's not include names in the wiki, instead use the @ to tag them by username. Better yet, use @ to tag a [GitHub team](https://github.com/orgs/ccao-data/teams).

File: How-To/Connect-to-AWS-Resources.md [hunk 2]
```diff
@@ -142,19 +143,20 @@ As with R, enabling [unload](https://laughingman7743.github.io/PyAthena/pandas.h
...
 3. Make sure [Java SE Development Kit](https://www.oracle.com/java/technologies/downloads/) is installed.
 4. Create a file called `athena.properties` in `C:\Users\$USER\Documents\My Tableau Repository\Datasources` on Windows or `~/Users/$USER/Documents/My Tableau Repository/Datasources` on Mac with the following lines:
     ```
     workgroup=read-only-with-scan-limit
     MetadataRetrievalMethod=ProxyAPI
     ```
-3. Open Tableau and on the `Connect` sidebar under `To a Server`, navigate to `Amazon Athena`.
+    Add the same file to `C:\Users\$USER\Documents\My Tableau Prep Repository\Datasources` on Windows or `~/Users/$USER/Documents/My Tableau Prep Repository/Datasources` on Mac.
+3. Open Tableau and on the `Connect` sidebar under `To a Server`, navigate to `Amazon Athena`. On Tableau Prep, select the `+` next to `Connections` and navigate to `Amazon Athena`.
 4. Message a [core team](https://github.com/orgs/ccao-data/teams/core-team) member for the necessary server info and credentials. Tableau will not save the `Secret Access Key` field.
```
Comment (dfsnow) [inline How-To/Connect-to-AWS-Resources.md:160]: **nitpick:** Can we update the numbering here?

File: How-To/Connect-to-AWS-Resources.md [hunk 3]
```diff
@@ -64,6 +64,7 @@ To setup and use `noctua` in an R project:
     AWS_REGION=us-east-1
     AWS_ATHENA_S3_STAGING_DIR=$RESULTS_BUCKET
     ```
+    Make sure that `$RESULTS_BUCKET` begins with "s3://"
```
Comment (dfsnow) [inline How-To/Connect-to-AWS-Resources.md:67]:
```suggestion
    Make sure that `$RESULTS_BUCKET` begins with `s3://`.
```

**Review by dfsnow — APPROVED** [review]
LGTM 👍

**Pull request merged**